packages/server/assets/core/typedefs/projects.graphql

```diff
@@ -12,6 +12,14 @@ enum ProjectVisibility {
   PUBLIC
 }
 
+"""
+Visbility without the "discoverable" option
+"""
+enum SimpleProjectVisibility {
```

Comment:
Still leaving the old enum and accepting it in create/update for backwards compatibility. If consumers send in "discoverable"/"public" as a value, it'll just be converted to UNLISTED behind the scenes.